bricks/template/__brick__/{{project_name.snakeCase()}}/README.md

```diff
@@ -18,29 +12,34 @@ Clone the repository
 ### Setup
 
 - Create these `.env` files in the root directory according to the flavors and add the required
-  environment variables into them. The example environment variable is in `.env.sample`.
+environment variables. The example environment variable is in `.env.sample`.
```

Comment:
I think we should not enter new line here 🤔

Reply:
It's just auto-formatting from VSCode. The final result looks good 🤷‍♂️ 

<img width="455" alt="image" src="https://github.com/nimblehq/flutter-templates/assets/16315358/d4921ab3-42b6-4481-8e13-32f4a45b0c99">